### DISCUSSION TITLE
Unity Barracuda | silero_vad.onnx

### DISCUSSION THREAD

**Discussion by andreobvio** [snakers4/silero-vad#288] [General]
Hi guys, I am trying to run the model in onnx format in Unity's Barracuda, however, as soon as I import the [silero_vad.onnx] to Barracuda it detects some problems in the model. Has anyone tried this?

I could just treat errors as warnings...

<img width="1085" alt="Screenshot 2023-01-20 at 10 30 22" src="https://user-images.githubusercontent.com/114940629/213706355-806e2640-96fc-4fdf-890f-2c6dcd817afa.png">
<img width="763" alt="Screenshot 2023-01-20 at 10 31 49" src="https://user-images.githubusercontent.com/114940629/213706583-4cc1ec33-9b0b-47f8-a5e3-3f39ecbee126.png">

So I could load it but then I get the warning:

<img width="667" alt="Screenshot 2023-01-20 at 10 53 41" src="https://user-images.githubusercontent.com/114940629/213713545-a90c15d1-3f83-4f08-872b-5a2cc96c3c5f.png">

how I load it:
<img width="621" alt="Screenshot 2023-01-20 at 10 54 21" src="https://user-images.githubusercontent.com/114940629/213713671-28164635-e212-4f20-998c-a408b17a5cb4.png">

**Comment by cochmedia** [comment]
hello, have you found a solution?
